### PR TITLE
IfcSpatialZone Rail Import BIMQ 2021-07-21T07:29:15.4359799Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcSpatialZone

### CODE REVIEW

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTensioningSectionGeneral/DocPropertySet.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayTensioningSectionGeneral" UniqueId="080715c0-01c0-25a8-6f15-d9cc7ab1f55e" ApplicableType="IfcSpatialZone/RESERVATION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTensioningSectionGeneral/DocPropertySet.xml:2]: what is the reasoning for this being seperate to Pset_SpatialZoneTypeReservationRailwayTensioningSection?

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
Clarification on the Application of Spatial Zone reservations before i can fully comment

comments on the definition of a Spatial Zone QTO can be addressed now.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneTypeReservationRailwayTensioningSection/DocPropertySet.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpatialZoneTypeReservationRailwayTensioningSection" UniqueId="60300302-e48a-083b-18f0-8153c7ce3195" ApplicableType="IfcSpatialZone/RESERVATION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="EndMileage_1CEznB3T6x9TP08LRzK7q" />
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneTypeReservationRailwayTensioningSection/DocPropertySet.xml]: They are removed

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneTypeReservationRailwayTensioningSection/DocPropertySet.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpatialZoneTypeReservationRailwayTensioningSection" UniqueId="60300302-e48a-083b-18f0-8153c7ce3195" ApplicableType="IfcSpatialZone/RESERVATION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneTypeReservationRailwayTensioningSection/DocPropertySet.xml]: I currently removed this Pset from this pull request

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpatialZoneOccuranceReservation" UniqueId="1831d08e-3c82-39e0-1a5f-4ccc322a7019" ApplicableType="IfcSpatialZone/RESERVATION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml]: This Pset is removed

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	<Properties>
+		<DocProperty xsi:nil="true" href="Height_2AzZDvvKPFDeOW8FDtpjKW" />
+		<DocProperty xsi:nil="true" href="Length_24pOWnHJn1RAC_Ic4zRseP" />
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml]: I agree, so they are added as quantities: https://github.com/bSI-InfraRoom/IFC-Specification/pull/181/commits/2055cd543ecc08e3da57a51e57eaa731a1ef449b

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+		<DocProperty xsi:nil="true" href="OccupiedHeight_0j9GoeSoLCH8nQA9XC2Tdz" />
+		<DocProperty xsi:nil="true" href="OccupiedLength_1uWEYQW9bDneFWlTG0fIZB" />
+		<DocProperty xsi:nil="true" href="OccupiedWidth_2moSva2gP828YdkUCpk36" />
```
[the remark this answers is not in the document]
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpatialZoneOccuranceReservation/DocPropertySet.xml]: I finally removed them: https://github.com/bSI-InfraRoom/IFC-Specification/pull/181/commits/93c1e92927276edb6aab3528fb0fea35676a86ed

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTensioningSectionGeneral/DocPropertySet.xml (hunk 1 above)
Comment (czapplitec) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTensioningSectionGeneral/DocPropertySet.xml]: I moved this Pset out of this pull request. I consider it as a backlog issue

**Review by AlexBrad1eyCT — APPROVED** [review]
well that got alot smaller! LFTM

**Pull request merged**